tests/base.py

```diff
@@ -86,6 +91,18 @@ def stopServer():
     """
     cherrypy.engine.exit()
     mockSmtp.stop()
+    dropAllTestDatabases()
+
+
+def dropAllTestDatabases():
+    """
+    Unless otherwise requested, drop all test databases.
+    """
+    if 'keepdb' not in os.environ.get('EXTRADEBUG', '').split():
```

Comment:
Should this function be generalized and parameterized with `usedDBs`, and only called in the case of `keepdb` (from `stopServer`)?